media/image.go

```diff
@@ -0,0 +1,22 @@
+package media
+
+import "fmt"
+
+const (
+	// ImageTypeJpeg JPEG format
+	ImageTypeJpeg string = "jpeg"
```

Comment:
Why not do it like this:

```
type ImageType string
const (
        // ImageTypeJpeg JPEG format
   	ImageTypeJpeg ImageType = "image/jpeg"

        // ImageTypePng PNG format
	ImageTypePng ImageType = "image/png"
)
```
It will make things a bit cleaner elsewhere in your code, since you a) now have the full mime string b) you have control over the type, so you don't need to worry about rogue strings.

Reply:
Good point. I was worried about supporting new image types if we added them, but we'd need to update the parsing code anyway, so might as well make this change. Done in 6d51d82